avro-serializer/src/test/java/io/confluent/kafka/serializers/KafkaAvroSerializerTest.java

```diff
@@ -46,7 +47,7 @@
 
   public KafkaAvroSerializerTest() {
     schemaRegistry = new LocalSchemaRegistryClient();
-    avroSerializer = new KafkaAvroSerializer(schemaRegistry);
+    avroSerializer = new KafkaAvroSerializer(schemaRegistry, GenericDatumWriter.class);
```

Comment:
Since this constructor is intended only for tests anyway, why not leave out the extra param and just have it default to the right thing? Even if someone constructs it directly, they could always use configure() to override the value, right?